RecoParticleFlow/PFProducer/plugins/importers/GeneralTracksImporterWithVeto.cc

```diff
@@ -123,7 +128,8 @@ void GeneralTracksImporterWithVeto::importToBlock(const edm::Event& e,
                               (pfmu_->hasValidTrack(muonref, false) && PFMuonAlgo::isMuon(muonref)));
     }
     if (thisIsAPotentialMuon ||
-        PFTrackAlgoTools::goodPtResolution(pftrackref->trackRef(), DPtovPtCut_, NHitCut_, useIterTracking_, debug_)) {
+        PFTrackAlgoTools::goodPtResolution(
+            pftrackref->trackRef(), DPtovPtCut_, NHitCut_, useIterTracking_, trackQuality_, debug_)) {
       trkElem = new reco::PFBlockElementTrack(pftrackref);
       if (thisIsAPotentialMuon && debug_) {
         std::cout << "Potential Muon P " << pftrackref->trackRef()->p() << " pt " << pftrackref->trackRef()->p()
```

Comment:
Also in this class the ```debug_``` parameter is not needed: just replace this line with a corresponding ```LogDebug``` (the same as already done in ```PFTrackAlgoTools.cc```) and remove all references to the ```debug_``` parameter in the code.